softee220/naverent_crawling.py

```
@@ -0,0 +1,178 @@
+
+# In[17]:
+
+
+from collections import Counter
+import re
+
+# 불용어 리스트 (한국어)
+korean_stopwords = set(['이', '그', '저', '것', '들', '다', '을', '를', '에', '의', '가', '이', '는', '해', '한', '하', '하고', '에서', '에게', '과', '와', '너무', '잘', '또','좀', '호텔', '아주', '진짜', '정말'])
+
+# 모든 리뷰를 하나의 문자열로 결합
+all_reviews_text = " ".join(reviews)
```

Ruff (0.13.2)

Undefined name `reviews`

(F821)